### ISSUE TITLE
302 to unpkg/jsDeliver when build failed?

### ISSUE THREAD

**Issue by ClassicOldSong** [Rich-Harris/packd#42]
Is it better to return a 302 to unpkg/jsDeliver when a package fails to build? Packages requires a custom build configuration like https://bundle.run/ef.js@0.6.2 cannot be handled correctly. Failed builds will return a 500 error which is not very user friendly. 302 to a live location could make a higher availability.

**Comment by xtuc** [comment]
I agree with your point, also the code `500` isn't very load balancer friendly.

> return a 302 to unpkg/jsDeliver

Bundle.run doesn't return anything to unpkg/jsDeliver. You mentioned that for the comparison?

> 302 to a live location could make a higher availability

What do you mean by live location? Bundle.run only allows you to build and fallbacking to unpkg/jsDeliver would fail in many cases because the client expected a bundled version.

**Comment by ClassicOldSong** [comment]
Many packages actually have a bundled version.

Here this site uses bundle.run https://codepan.net/boilerplate/es-import to handle imports but some packages failed to load.

**Comment by xtuc** [comment]
I understand, but looking at the requests to bundle.run everything seems fine:
![screenshot from 2018-02-19 18-19-07](https://user-images.githubusercontent.com/1493671/36389983-847eb6b4-15a1-11e8-8934-a5e328565a8b.png)

I'm not able to see the error on codepan, could you please send me it?

**Comment by ClassicOldSong** [comment]
Try this in the code panel:

``` javascript
import * as ef from 'ef.js'
import backbone from 'backbone'
```

**Comment by xtuc** [comment]
> backbone

jQuery is `require`'d here https://github.com/jashkenas/backbone/blob/master/backbone.js#L26 and not defined in the dependencies. That probably works in Nodejs but it seems that we can't bundle it.

**Comment by xtuc** [comment]
> ef.js

The error on bundle.run isn't super useful but I think it's the spread syntax. It works fine when you build it because it uses rollup with your config. I'm not sure how to do to be honest in that case.

**Comment by ClassicOldSong** [comment]
Yes I'm talking about custom build config. Maybe just run the build command provided by the package and monitor file changes or check the dist folder beforehand?
